Makefile

```diff
@@ -10,7 +10,7 @@ PROJECT_NAME := $(shell basename "$(PWD)")
 # Go related variables.
 GO_FILES := $(shell find . -type f -name '*.go' | grep -v vendor)
 GOBASE := $(shell pwd)
-GOBIN := $(GOBASE)/bin
+GOBIN := $(GOBASE)/dist
```

Comment:
Why are you changing the folder??

Reply:
Goreleaser already uses `dist` as a distribution release folder. Now we have two different folders, one in CI another one in the Makefile. I'm trying to be standardized and keep only one. And also, the `bin` folder is used to put the binaries you want your machine to execute in the runtime, `dist` is more appropriate to put the build release binaries